src/Logger/LoggerFactory.php

```diff
@@ -50,6 +51,18 @@ public function create(): Logger
         return new Logger($innerLogger);
     }
 
+    public function createEvent(Event $event): Logger
```

Comment:
Is whole `Event` object needed where? Passing new `output_log_file` should be enough.